### DISCUSSION TITLE
Sell level 70 Northrend food and drink in Shattrath

### DISCUSSION THREAD

**Discussion by Xorrak32** [chromiecraft/chromiecraft#5887] [Suggestions]
I think it's time for Shattrath innkeepers in Scryers and Aldor to start selling the level 70 northrend food and drinks. Many people don't know that the main city sell them so you have to tell people to go buy those instead. It really helps in dungeons and it's also just a logical step at this point in my opinion.

**Comment by balleny** [comment]
the logical step would be to remove them as they shouldnt be accessible yet.
Shattrath is unique in some sense, as it doesnt feature any class trainers, auction house weapon trainer, training dummies and no wotlk food (neither now nor in a later stage)

**Reply by Xorrak32** [reply]
This is a wrath client with wrath talents. Removing those drinks will be counter-productive. It already takes too long with current drinks. We have more mana then what you would have back in tbc and it is only going to get worse when we're starting to get raid gear. Tier 6 we will probably have up to 20k mana (I have over 14k alredy with buffs on my shaman). 
Again, this is not a 100% blizzlike server. The chormie devs themselves have made some liberties to the server. Removing these would be foolish and anger people, not help them.

So yet again, placing this drink and food where it is more widely available should be there. Let's not waste more time sitting down and waiting for people to get mana. Let's get dungeons and raids done.

**Reply by balleny** [reply]
the best solution, if there are any changes planned, is to introduce the 3 or 5 percent health and mana food which is available during world events as regular food from vendors for the costs of the highest level (75 or 80) costs of regular food.
Why the costs from the higher level? cause these foods would be an optional, beneficial factor for max level players mostly AND if they select the correct one, it would be health and mana at the same time.
5% option https://www.wowhead.com/wotlk/item=21215/graccus-mince-meat-fruitcake
4% option https://www.wowhead.com/wotlk/item=23172/refreshing-red-apple

**Reply by Xorrak32** [reply]
The fruitcake could definitely be a solution. It is even better than seal whey the more gear you get.
